portfolio/src/main/webapp/comment.js

```diff
@@ -13,22 +13,41 @@
 // limitations under the License.
 
 /**
-*fetches current comments from server and puts it to HTML
+* checks if user is logged in
+*/
+function checkUser() {
+  fetch('/check-user').then((response)=>
+    response.json()).then((isLoggedIn)=> {
+    console.log(isLoggedIn);
+    if (isLoggedIn == false) {
+      console.log('ok');
+      document.getElementById('addcontent').placeholder = 'log in first';
+      document.getElementById('addcontent').disabled = true;
+      document.getElementById('color').disabled = true;
+      document.getElementById('font_size').disabled = true;
+      document.getElementById('button').disabled = true;
```

Comment:
I guess this is the code you were referring to earlier. I think this works well enough :)
As I mentioned in the long comment above, this doesn't actually prevent the user from sending an 'add comment' request. But this isn't a problem if you handle things safely on the backend.